test/e2e/constants/onboarding.py

```diff
@@ -5,7 +5,7 @@
 class OnboardingMessages(Enum):
     WRONG_LOGIN_LESS_LETTERS = 'Display Names must be at least 5 character(s) long'
     WRONG_LOGIN_SYMBOLS_NOT_ALLOWED = 'Invalid characters (use A-Z and 0-9, hyphens and underscores only)'
-    WRONG_PASSWORD = 'Password must be at least 10 characters long'
+    WRONG_PASSWORD = 'Minimum 10 character(s)'
```

Comment:
This still does not inform about the max length (100).

#16239

Reply:
thats verification from test for error string that is less than 10 . Whatever apps says i just verify

Reply:
This is a test; the app will inform the user accordingly; see the first screenshot in this PR